berkeley-mobile/Utils/UIViewController+Extensions.swift

```diff
@@ -28,22 +28,34 @@ extension UIViewController {
     
     // Presents an alert with multiple options and completion handler
     public func presentAlertLinkUrl(title: String, message: String, options: String..., website_url: URL) {
-        let alertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        
+        var alertVC: AlertView!
+        
         for (index, option) in options.enumerated() {
             if (index == 0) {
-                alertController.addAction(UIAlertAction.init(title: option, style: .cancel))
+                alertVC = AlertView(headingText: title, messageText: message, action1Label: option, action1Color: Color.AlertView.secondaryButton, action1Completion: {
+                    self.dismiss(animated: true, completion: nil)
+                }, action2Label: "", action2Color: UIColor.clear, action2Completion: {return}, withOnlyOneAction: true)
             } else {
-                alertController.addAction(UIAlertAction.init(title: option, style: .default, handler: { _ in
+                
+                alertVC = AlertView(headingText: title, messageText: message, action1Label: "Cancel", action1Color: Color.AlertView.secondaryButton, action1Completion: {
+                    self.dismiss(animated: true, completion: nil)
+                }, action2Label: "Yes", action2Color: Color.ActionButton.background, action2Completion: {
```

Comment:
this is a bug/issue from before, but we should dismiss the alert after opening the url.
right now if the user hits "Yes", the url opens but when they go back to the app the AlertView is still there